### ISSUE TITLE
Semi-relative paths are corrupted

### ISSUE THREAD

**Issue by renehamburger** [SoominHan/import-sorter#13]
Thanks for your extension!

We're using path aliases like `app` throughout, but on rare occasions we need to access files one level up. But such a 'semi-relative' import statement like
```ts
import { addMatchers } from 'app/../testing';
```
is converted by your extension to
```ts
import { addMatchers } from 'testing';
```
and thus breaks.

**Comment by mike-co** [comment]
Sure, will fix that

**Comment by renehamburger** [comment]
Thanks!

**Comment by mike-co** [comment]
Should be fixed in 2.0.1

**Comment by renehamburger** [comment]
Thanks a lot! It's working now.
